magma/backend/coreir/coreir_compiler.py

```diff
@@ -107,8 +107,12 @@ def compile(self):
             self._compile_verilog_epilogue()
             if self.opts.get("generate_symbols", False):
                 coreir_symbol_table = _get_coreir_symbol_table(self.basename)
-                master = MasterSymbolTable([symbol_table, coreir_symbol_table])
-                result["master_symbol_table"] = master
+                if self.opts.get("dont_merge_symbol_tables", False):
```

Comment:
Perhaps we can use "merge_symbol_tables=False" instead of "dont_merge_symbol_tables=True" (just a shorter option name)?